app/graphql/inputs/enabled_dynamic_field_input.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Inputs
+  class EnabledDynamicFieldInput < Types::BaseInputObject
```

Comment:
On the ruby-graphql Input Object page (https://graphql-ruby.org/type_definitions/input_objects), they show an example subclass that ends in "Attributes" (Types::PostAttributes), but later in the page also show an example subclass that ends in "Input" (Types::DateRangeInput).  We've used both forms in HYC3, but I'd like to recommend that we standardize on one.  If you agree, do you prefer one convention over the other?

Reply:
Hmm... on the one hand the document illustrates a certain lack of common practice :) But some convention makes it easier to focus your mental energy. If we were going to standardize, I think I prefer *Input - which I think also aligns with the generated bits of the schema, right?

Reply:
*Input sounds good to me, and you're right about it aligning well.  If okay, I'll create a ticket as a TODO for later?

The other thing I like about having "Input" in the name is that because an input object can never be returned as a result, it'll help us to quickly differentiate between any similarly-named Types that actually can be returned as a result (e.g. when searching through files in an IDE).

Reply:
Ticket created!